evaluation/tests/test_evaluation_scripts.sh

```diff
@@ -96,12 +100,21 @@ pipeline_microbenchmarks=(
 execute_pash_and_check_diff() {
     TIMEFORMAT="%3R" # %3U %3S"
     if [ "$DEBUG" -eq 1 ]; then
-        { time "$PASH_TOP/pa.sh" $@ ; } 1> "$pash_output" 2> >(tee -a "${pash_time}" >&2) &&
-        diff -s "$seq_output" "$pash_output" | head | tee -a "${pash_time}" >&2
+        if [ "$test_mode" == "bash" ]; then
```

Comment:
I feel that we can just modify the file to run tests both with --bash and without (modifying the configurations array above).